dist/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>The Food App</title>
+    <script defer src="main.js"></script></head>
+    <body>
+        <div class="container">
+            <header>
```

Comment:
- [ ] As requested by the previous code reviewer, which is also a requirement for this project, please ensure the implementation of your comment popup follows the [design](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md). Please make sure to follow the design for the comment popup i.e the close button the comments position